src/main/steps/tasklistresponse/international-factors/InternationalFactorsMapper.ts

```diff
@@ -1,116 +1,72 @@
+    anotherPersonOrderOutsideEnWl: jurisdiction,
+    anotherPersonOrderOutsideEnWlDetails: iFactorsJurisdictionProvideDetails,
+    anotherCountryAskedInformation: request,
+    anotherCountryAskedInformationDetaails: iFactorsRequestProvideDetails,
+  });
+
+  if (start === YesOrNo.NO) {
+    delete internationalElementsRequest.childrenLiveOutsideOfEnWlDetails;
   }
 
-  return respondent;
-};
-
-export const getInternationalFactorsDetails = (respondent: Respondent, req: AppRequest): Partial<CaseWithId> => {
-  const tempValue = {};
-
-  if (respondent?.value?.response?.citizenInternationalElements?.childrenLiveOutsideOfEnWl === YesOrNo.NO) {
-    Object.assign(tempValue, { start: YesOrNo.NO, iFactorsStartProvideDetails: '' });
-  }
-  if (respondent?.value?.response?.citizenInternationalElements?.childrenLiveOutsideOfEnWl === YesOrNo.YES) {
-    Object.assign(tempValue, {
-      start: YesOrNo.YES,
-      iFactorsStartProvideDetails:
-        respondent?.value?.response?.citizenInternationalElements?.childrenLiveOutsideOfEnWlDetails,
-    });
-  }
-  if (respondent?.value?.response?.citizenInternationalElements?.parentsAnyOneLiveOutsideEnWl === YesOrNo.NO) {
-    Object.assign(tempValue, { parents: YesOrNo.NO, iFactorsParentsProvideDetails: '' });
+  if (parents === YesOrNo.NO) {
+    delete internationalElementsRequest.parentsAnyOneLiveOutsideEnWlDetails;
   }
-  if (respondent?.value?.response?.citizenInternationalElements?.parentsAnyOneLiveOutsideEnWl === YesOrNo.YES) {
-    Object.assign(tempValue, {
-      parents: YesOrNo.YES,
-      iFactorsParentsProvideDetails:
-        respondent?.value?.response?.citizenInternationalElements?.parentsAnyOneLiveOutsideEnWlDetails,
-    });
-  }
-  if (respondent?.value?.response?.citizenInternationalElements?.anotherPersonOrderOutsideEnWl === YesOrNo.NO) {
-    Object.assign(tempValue, { jurisdiction: YesOrNo.NO, iFactorsJurisdictionProvideDetails: '' });
-  }
-  if (respondent?.value?.response?.citizenInternationalElements?.anotherPersonOrderOutsideEnWl === YesOrNo.YES) {
-    Object.assign(tempValue, {
-      jurisdiction: YesOrNo.YES,
-      iFactorsJurisdictionProvideDetails:
-        respondent?.value?.response?.citizenInternationalElements?.anotherPersonOrderOutsideEnWlDetails,
-    });
-  }
-  if (respondent?.value?.response?.citizenInternationalElements?.anotherCountryAskedInformation === YesOrNo.NO) {
-    Object.assign(tempValue, { request: YesOrNo.NO, iFactorsRequestProvideDetails: '' });
+
+  if (jurisdiction === YesOrNo.NO) {
+    delete internationalElementsRequest.anotherPersonOrderOutsideEnWlDetails;
   }
-  if (respondent?.value?.response?.citizenInternationalElements?.anotherCountryAskedInformation === YesOrNo.YES) {
-    Object.assign(tempValue, {
-      request: YesOrNo.YES,
-      iFactorsRequestProvideDetails:
-        respondent?.value?.response?.citizenInternationalElements?.anotherCountryAskedInformationDetaails,
-    });
+
+  if (request === YesOrNo.NO) {
```

Comment:
what does request mean? What data exactly does the variable hold?

Reply:
request is the name of the radio button for the international-factors/request page and it holds the value of that radio button